### DISCUSSION TITLE
How to update chatterino to `beta` version from `stable`?

### DISCUSSION THREAD

**Discussion by MrPandir** [Chatterino/chatterino2#5324] [Q&A] [answered]
I know that you can download a release from GitHub and replace the program with it, but can this be done from Chatterino? 🤔

I found the Receive beta updates option in the settings, but I don't understand how it works.
I assume that it should offer me to install the update, just like it works with the stable versions - is that correct?
If that's the case, then this is not happening.

**Comment by pajlada** [comment, marked as answer]
The update checker will only run on startup - once you've enabled the "Receive beta updates" option, you should be prompted for an update next time you restart the application

**Reply by MrPandir** [reply]
Okay, this automatically downloads the file and closes chatterino.

I'm wondering why Chatterino can't do the file replacement itself and restart itself? Is this related to the operating system?

**Reply by pajlada** [reply]
The windows situation does that, but macOS & Linux are a bit special in that regard yeah. Preferably we'd just distribute the macOS version through brew with a beta branch or something.
